suites/pacific/cephadm/tier-0.yaml

```diff
@@ -0,0 +1,122 @@
+#===============================================================================================
+#-------------------------------------
+#-----    Smoke Test suite     ------
+#-------------------------------------
+# Conf: conf/pacific/cephadm/tier-0.yaml
+# Smoke test cases for
+#    - Bootstrap
+#    - Host management
+#    - Ceph role Service deployment,
+#    - Configure client for RGW and RBD systems
+#
+#===============================================================================================
+tests:
+  - test:
+      abort-on-fail: true
+      desc: Install software pre-requisites for cluster deployment.
+      config:
+        is_production: true
```

Comment:
Shouldn't this be set to `false`? This being a suite executed against every development build I believe the key should not be part of it